src/main/webapp/app/entities/quiz/quiz-exercise.model.ts

```diff
@@ -51,6 +51,7 @@ export class QuizExercise extends Exercise {
 
     public isActiveQuiz?: boolean;
     public isPracticeModeAvailable?: boolean;
+    public isEditable?: boolean;
```

Comment:
Add default value and comment in constructors as with the other ones maybe?

Reply:
Thanks for the suggestion. I added the default value and comment in the constructor.